module/VuFind/config/module.config.php

```diff
@@ -663,6 +663,10 @@
 
 // Define record view routes -- route name => controller
 $recordRoutes = [
+    // For legacy (1.x/2.x) compatibility (defined first so that RouteMatch uses the
+    // latter 'record' route as the matched route):
+    'vufindrecord' => 'Record',
```

Comment:
Did you dig into the history of this at all? I can't remember exactly why this is here, but my suspicion is that it may be old enough that we can simply remove it and put a note in the changelog. If you haven't taken the time to investigate the history at all, let me know, and I'll refresh my memory and elaborate here once I know more. Unless I'm forgetting something important, I can only theorize that this could impact either old saved searches or old custom code, and in both cases, I would hope that at this point there should be a way to bring things up to date and abandon this old configuration.

Reply:
I didn't, but now that I did I believe it's quite safe to remove. I can't recall ever seeing it used.